src/sentry/release_health/metrics.py

```diff
@@ -551,7 +551,7 @@ def check_has_health_data(
         if now is None:
             now = datetime.now(pytz.utc)
 
-        start = now - timedelta(days=3)
+        start = now - timedelta(days=90)
```

Comment:
I found this while going over all `should_compare` instances.. @RaduW was there a specific reason to use only 3 days in the metrics implementation vs. 90 days in the sessions implementation?

https://github.com/getsentry/sentry/blob/0afb8502a31ab13b05653538f91fb9e80a9d3227/src/sentry/snuba/sessions.py#L121